qiskit_dynamics/solvers/solver_classes.py

```diff
@@ -224,97 +302,151 @@ def solve(
 
         Additional Information:
 
-            The behaviour of this method is impacted by the input type of ``y0``:
+        The behaviour of this method is impacted by the input type of ``y0``:
 
-             * If ``y0`` is an ``Array``, it is passed directly to
-                :func:`~qiskit_dynamics.solve_lmde` as is. Acceptable array shapes are
-                determined by the model type and evaluation mode.
-             * If ``y0`` is a subclass of :class:`qiskit.quantum_info.QuantumState`:
+         * If ``y0`` is an ``Array``, it is passed directly to
+            :func:`~qiskit_dynamics.solve_lmde` as is. Acceptable array shapes are
+            determined by the model type and evaluation mode.
+         * If ``y0`` is a subclass of :class:`qiskit.quantum_info.QuantumState`:
```

Comment:
An idea for this would be to put a table somewhere to make the solver modes easier to see eg along lines of

```
| y0 | Model | yf | Method / Details / Description |
| --- | --- | --- |
| Array, np.ndarray | Any | Same as y0 | solve_lmde ... |
| Statevector | HamiltonianModel | Statevector | ... |
| Statevector | LindbladModel | DensityMatrix | ... |
| DensityMatrix | Hamiltonian, Lindblad | DensityMatrix  | ... |
| Operator | Hamiltonian | Operator | .... |
| Operator | Lindblad | Choi/SuperOp? | ... |
| QuantumChannel | Hamiltonian, Lindlbad | Choi/SuperOp? | ... |
```

Reply:
Yeah agreed - just made this change.